themes/prism-one-dark.css

```diff
@@ -248,5 +244,186 @@ pre[class*="language-"] {
+/* Space out all buttons and move them away from the right edge of the code block */
+div.code-toolbar > .toolbar > .toolbar-item {
+	margin-right: 0.4em;
+}
+
+/* Styling the buttons */
+div.code-toolbar > .toolbar > .toolbar-item > button,
+div.code-toolbar > .toolbar > .toolbar-item > a,
+div.code-toolbar > .toolbar > .toolbar-item > span {
+	background: hsl(220, 13%, 26%);
+	color: hsl(220, 9%, 55%);
+	padding: 0.1em 0.4em;
+	border-radius: 0.3em;
+}
+
+div.code-toolbar > .toolbar > .toolbar-item > button:hover,
+div.code-toolbar > .toolbar > .toolbar-item > button:focus,
+div.code-toolbar > .toolbar > .toolbar-item > a:hover,
+div.code-toolbar > .toolbar > .toolbar-item > a:focus,
+div.code-toolbar > .toolbar > .toolbar-item > span:hover,
+div.code-toolbar > .toolbar > .toolbar-item > span:focus {
+	background: hsl(220, 13%, 28%);
+	color: hsl(220, 14%, 71%);
+}
+
+/* Line Highlight plugin overrides */
+.line-highlight {
+	background: hsla(220, 100%, 80%, 0.04);
+}
+
+/* Default line numbers in Line Highlight plugin */
+.line-highlight:before,
+.line-highlight[data-end]:after {
+	background: hsl(220, 13%, 26%);
+	color: hsl(220, 14%, 71%);
+	padding: 0.1em 0.6em;
+	border-radius: 0.3em;
+	box-shadow: 0 2px 0 0 rgba(0, 0, 0, 0.2); /* same as Toolbar plugin default */
+}
+
+/* Hovering over a linkable line number (in the gutter area) */
+pre[id].linkable-line-numbers span.line-numbers-rows > span:hover:before {
+	background-color: hsla(220, 100%, 80%, 0.04);
+}
+
+/* Line Numbers and Command Line plugins overrides */
+/* Line separating gutter from coding area */
+.line-numbers .line-numbers-rows,
+.command-line-prompt {
+	border-right-color: hsla(220, 14%, 71%, 0.15);
+}
+
+/* Stuff in the gutter */
+.line-numbers-rows > span:before,
+.command-line-prompt > span:before {
+	color: hsl(220, 14%, 71%); /* would prefer --syntax-gutter but it fails WCAG AA */
+}
+
+/* Match Braces plugin overrides */
+/* Note: Outline colour is inherited from the braces */
+.rainbow-braces .token.punctuation.brace-level-1,
+.rainbow-braces .token.punctuation.brace-level-5,
+.rainbow-braces .token.punctuation.brace-level-9 {
+	color: hsl(355, 65%, 65%);
+}
+
+.rainbow-braces .token.punctuation.brace-level-2,
+.rainbow-braces .token.punctuation.brace-level-6,
+.rainbow-braces .token.punctuation.brace-level-10 {
+	color: hsl(95, 38%, 62%);
+}
+
+.rainbow-braces .token.punctuation.brace-level-3,
+.rainbow-braces .token.punctuation.brace-level-7,
+.rainbow-braces .token.punctuation.brace-level-11 {
+	color: hsl(207, 82%, 66%);
+}
+
+.rainbow-braces .token.punctuation.brace-level-4,
+.rainbow-braces .token.punctuation.brace-level-8,
+.rainbow-braces .token.punctuation.brace-level-12 {
+	color: hsl(286, 60%, 67%);
+}
+
+/* Diff Highlight plugin overrides */
+/* Taken from https://github.com/atom/github/blob/master/styles/variables.less */
+pre.diff-highlight > code .token.deleted:not(.prefix),
+pre > code.diff-highlight .token.deleted:not(.prefix) {
```

Comment:
This rule has the same specificity as the rule of the plugin. This means that the order of the stylesheets (this one and the plugin one) which background we get. This is bad because we don't necessarily control that order.

To fix this, we need to make sure that this rule has a higher specificity. The easiest way to do that is to repeat one of the classes. E.g.:

```css
pre.diff-highlight > code .token.token.deleted:not(.prefix),
pre > code.diff-highlight .token.token.deleted:not(.prefix) { ... }
```

Looks a bit silly but it gets the job done. 

I show an example with Diff Highlight below but pretty much all rules this problem. This fix has to be applied to all rules that you copied from the plugins' CSS.

Example:
(`prism-dark.css` is this theme in this example.)
![image](https://user-images.githubusercontent.com/20878432/133150465-2d0b71e0-f148-4a62-b285-4cfaa33796fb.png)

Reply:
Ah okay! I didn't consider that we (or the users) may not have full control over the order of the stylesheets, thanks for pointing it out. I think adding another `.token` selector is a little silly, I'll see if I can use something more semantic, but if that doesn't work out, I'll use `.token`.

Reply:
Also, as a note, I referenced a few other themes' stylesheets when writing these overrides. I can't remember exactly which ones have overrides, but I can look through them again and increase their selectors' specificities where needed (in another PR)!